painter.go

```diff
@@ -70,11 +70,16 @@ func GeneratePoster(posterDetails Document) {
 	surface.MoveTo(x, y)
 	surface.ShowText(text)
 
+	surface.MoveTo(x-20, y)
+	surface.SelectFontFace("FontAwesome", cairo.FONT_SLANT_NORMAL, cairo.FONT_WEIGHT_NORMAL)
```

Comment:
Change : Instead of using FontAwesome, we could better use some SVG icon.
Reason  : Not having FontAwesome installed could result in an awkward poster.

Reply:
Have a look here for pointers, https://github.com/webkul/vivid/tree/master/icons

Reply:
Then we need to ship the icons along with the binary then, with the font you can tell the target user to install the font.

Reply:
I was thinking of embedding them into the binary.